tomatic/ui/src/services/api.js

```diff
@@ -1,58 +1,92 @@
-import m from 'mithril'
 import jsyaml from 'js-yaml'
 import Auth from './auth'
 import messages from './messages'
 
-const debugApi = false
+const debugApi = true
```

Comment:
What is the reason of debugging as default? Would be worth extract the flag to a configuration/env file?